Cargo.toml

```diff
@@ -12,12 +12,15 @@ cairo-lang-sierra-to-casm = "=2.5.0"
 cairo-proof-parser = { git = "https://github.com/Okm165/cairo-proof-parser", rev = "97a04bbee07330311b38d6f4cecfed3acb237626"}
 cairo-vm = "1.0.0-rc3"
 clap = { version = "4.3.10", features = ["derive"] }
+flate2 = "1.0.30"
 itertools = "0.13.0"
 num-bigint = "0.4.6"
+reqwest = { version = "0.12.5", features = ["blocking"] }
```

Comment:
If you only need these dependencies for `build.rs`, you can add them to `[build-dependencies]`

Reply:
Done